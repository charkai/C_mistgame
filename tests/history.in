place A12
history
place B9
place C12
history
place D11
place S1
place A21
place A1234
place S1
history
term
