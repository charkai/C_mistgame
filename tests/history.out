A12
A12B9C12
Invalid coordinate
Invalid coordinate
Occupied coordinate
A12B9C12D11S1
